pkgs/shells/fzf-git-sh/default.nix

```diff
@@ -39,12 +39,12 @@ stdenv.mkDerivation rec {
       -e "s,\buniq\b,${coreutils}/bin/uniq," \
       -e "s,\bcolumn\b,${util-linux}/bin/column," \
       -e "s,\bfzf-tmux\b,${fzf}/bin/fzf-tmux," \
-      -e "/display-message/!s,\bgit\b,${git}/bin/git,g" \
       -e "s,\bgrep\b,${gnugrep}/bin/grep," \
       -e "s,\bsed\b,${gnused}/bin/sed," \
       -e "/fzf-tmux/!s,\btmux\b,${tmux}/bin/tmux," \
       -e "s,\bxargs\b,${findutils}/bin/xargs," \
       -e "s,\bxdg-open\b,${xdg-utils}/bin/xdg-open," \
+      -e "/display-message\|fzf-git-\$o-widget\|^\( \)*url=\|\$remote_url =~ ^/!s,\bgit\b,${git}/bin/git,g" \
```

Comment:
Instead of `\( \)*url=\`, why not just use `\burl=`?

I think it would be better to remove the `^` from `\$remote_url =~ ^` (or at least escape it), since `^` is a special character, so this could cause confusion.